tests/resources/dspa-external.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: datasciencepipelinesapplications.opendatahub.io/v1alpha1
+kind: DataSciencePipelinesApplication
+metadata:
+  name: dspa-ext
```

Comment:
how is this different from dspa-ext in the /tests/resources/ folder? are we duplicated files ? can we just use `kustomization.yaml` to reference this instead of duplicating it?